### PR TITLE
 [MIG] purchase_triple_discount: Migration to 11.0

### PR DESCRIPTION
**Pull request by chienandalu**
superseeds these abandoned PRs: https://github.com/OCA/purchase-workflow/pull/595 and https://github.com/OCA/purchase-workflow/pull/611
- Integrate with supplierinfo_dicount
- readme parts

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
If CIs go well

**Review by HviorForgeFlow — APPROVED** [review]
Functional review
LGTM
:+1:

**Comment by pedrobaeza** [comment]
We are going to add the purchase.supplierinfo stuff here as well, so setting WIP.

**Comment by chienandalu** [comment]
Changes done to integrate with purchase.supplierinfo

**Review by pedrobaeza — APPROVED** [review]
Thanks for the extra tests on new features.

File: purchase_triple_discount/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    def _prepare_invoice_line_from_po_line(self, line):
+        vals = super(AccountInvoice,
+                     self)._prepare_invoice_line_from_po_line(line)
+        vals['discount2'] = line.discount2
```
Comment (nikul-serpentcs) [inline purchase_triple_discount/models/account_invoice.py:13]: @chienandalu I think, It's batter :smile: 
```
return {
    'discount2': line.discount2,
    'discount3': line.discount3
}
```

**Review by nikul-serpentcs — APPROVED** [review]
Otherwise Code Review LGTM :+1:

File: purchase_triple_discount/models/account_invoice.py (hunk 1 above)
Comment (chienandalu) [reply purchase_triple_discount/models/account_invoice.py:13]: No, what you propose is wrong. You're ignoring the values from the super()

Comment (nikul-serpentcs) [reply purchase_triple_discount/models/account_invoice.py:13]: Your Code Is Perfect But I suggesting

Typo Mistakes

val.update( {
    'discount2': line.discount2,
    'discount3': line.discount3
})
return val

Comment (pedrobaeza) [reply purchase_triple_discount/models/account_invoice.py:13]: Why it's better? You are writing 4 lines of code against 2 and no performance difference.

**Comment by legalsylvain** [comment]
Thanks a lot @chienandalu.

**Pull request merged**